doc/changes/changes_10.4.0.md

```diff
@@ -0,0 +1,15 @@
+# Virtual Schema Common JDBC 10.4.0, released 2023-03-10
+
+Code name: Validators for Adapter Properties
+
+## Summary
+
+This release adds validators for adapter properties enabling derived SQL dialects to simply add these validator to the existing chain.
+* `ImportProperty` checking the consistency of import and connection properties
```

Comment:
Small thing, would it make sense to suffix them with `Validator`, e.g, ImportProperyValidator?